tests/test_json_interface.c

```diff
@@ -25,16 +25,19 @@
 #include "../src/json_interface.h"
 
 /*----------------------------------------------------------------------------*/
-/*                                   Mocks                                    */
+/*                                  Macros                                    */
 /*----------------------------------------------------------------------------*/
-FILE *g_file_handle;
-
 #define TEST_FILE_NAME  "test.json"
 
 #define TEST1_ENTRY     "service1"
 #define TEST1_VALUE     "url1"
 
 /*----------------------------------------------------------------------------*/
+/*                                   Mocks                                    */
+/*----------------------------------------------------------------------------*/
+FILE *g_file_handle;
```

Comment:
This is temporary, will be changed in the next set of changes, moving all file handling to JSON interface.